src/screens/UnfreezeFunds/01-Amount.js

```diff
@@ -0,0 +1,334 @@
+/* @flow */
+import { BigNumber } from "bignumber.js";
+import useBridgeTransaction from "@ledgerhq/live-common/lib/bridge/useBridgeTransaction";
+import React, { useCallback, useMemo } from "react";
+import { View, StyleSheet, TouchableOpacity } from "react-native";
+import { SafeAreaView } from "react-navigation";
+import { connect } from "react-redux";
+import { compose } from "redux";
+import { translate, Trans } from "react-i18next";
+import i18next from "i18next";
+import type { NavigationScreenProp } from "react-navigation";
+import type { Account, Transaction } from "@ledgerhq/live-common/lib/types";
+
+import { getAccountUnit } from "@ledgerhq/live-common/lib/account";
+import { getAccountBridge } from "@ledgerhq/live-common/lib/bridge";
+
+import { accountAndParentScreenSelector } from "../../reducers/accounts";
+import colors from "../../colors";
+import { TrackScreen } from "../../analytics";
+import LText from "../../components/LText";
+import Button from "../../components/Button";
+import StepHeader from "../../components/StepHeader";
+import RetryButton from "../../components/RetryButton";
+import CancelButton from "../../components/CancelButton";
+import GenericErrorBottomModal from "../../components/GenericErrorBottomModal";
+import Info from "../../icons/Info";
+import CurrencyUnitValue from "../../components/CurrencyUnitValue";
+import TranslatedError from "../../components/TranslatedError";
+import CheckBox from "../../components/CheckBox";
+import Bandwidth from "../../icons/Bandwidth";
+import Bolt from "../../icons/Bolt";
+
+const forceInset = { bottom: "always" };
+
+type Props = {
+  account: Account,
+  parentAccount: ?Account,
+  navigation: NavigationScreenProp<{
+    params: {
+      accountId: string,
+      transaction: Transaction,
+    },
+  }>,
+};
+
+const UnfreezeAmount = ({ account, parentAccount, navigation }: Props) => {
+  const bridge = getAccountBridge(account, parentAccount);
+  const unit = getAccountUnit(account);
+
+  const {
+    transaction,
+    setTransaction,
+    status,
+    bridgePending,
+    bridgeError,
+  } = useBridgeTransaction(() => {
+    const { tronResources: { frozen: { bandwidth } = {} } = {} } = account;
```

Comment:
```js
const { tronResources } = account;
invariant(tronResources, "tron resources expected");
const { frozen: { bandwidth } } = tronResources;
```

all the things please =D

- this is more readable.
- this give better typing and avoid you do to all the `bandwidth && bandwidth.amount ` & co..